pom.xml

```diff
@@ -74,6 +74,9 @@
         <deploy.disabled>false</deploy.disabled>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+        <testcontainers.version>1.11.2</testcontainers.version>
+        <minio.version>6.0.6</minio.version>
+        <httpcomponents>4.5.2</httpcomponents>
```

Comment:
let's keep the same approach httpcomponents  -> httpcomponents.version

Reply:
of course. just miss word version